test_dir_1/owners.txt

```diff
@@ -0,0 +1,2 @@
+RiflerRick
```

Comment:
RiflerRick committed on a restricted repository: BB_MCS_Test belonging to Rajdeep Mukherjee(RiflerRick) authored by @RiflerRick in the file test_dir_1/owners.txt
Owners: @RiflerRick,,